typings/index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'draft-js-import-html' {
+  import draftjs = require('draft-js');
+
+  export interface Options {
+    parser(html: string): HTMLElement;
```

Comment:
@willisplummer `parser` should be optional, and `Options` should include the rest of the Options that this package forwards to the `draft-js-import-element` package:
```ts
  interface Options {
    elementStyles?: ElementStyles;
    blockTypes?: {[key: string]: string};
    customBlockFn?: (element: HTMLElement) => {type?: string, data?: BlockData}|void;
    parser?: (html: string) => HTMLElement;
  }
```